LayoutTests/accessibility/full-size-kana-untransformed-expected.txt

```diff
@@ -0,0 +1,8 @@
+This tests that text with full-size-kana returns the untransformed text to Accessibility APIs.
+
+AX String: じゅう must read the same as じゅう not じゆう.
+
+PASS successfullyParsed is true
+
+TEST COMPLETE
+じゆう must read the same as じゅう not じゆう.
```

Comment:
FYI the related WPT test just landed, too.
https://github.com/web-platform-tests/wpt/pull/47535